configs/config/debugging/pretrain/supervised/supervised_1gpu_vision_transformer_debug_integration_cutmix.yaml

```diff
@@ -19,7 +19,7 @@ config:
       TRANSFORMS:
         - name: RandomResizedCrop
           size: 224
-        - name: VisslRandAugment
+        - name: RandAugment
```

Comment:
we should rebase to master. This change should be essentially reverted after the rebase.